clowdapp.yaml

```diff
@@ -473,3 +473,6 @@ parameters:
 - name: FLOORIST_LOGLEVEL
   description: Floorist loglevel config
   value: 'INFO'
+- name: COMPLIANCE_DEPENDENCIES_SUFFIX
```

Comment:
I'd make it a singular `COMPLIANCE_DEPENDENCY_SUFFIX`

Reply:
Changed. Thank you